rasa/core/policies/ensemble.py

```diff
@@ -212,7 +206,8 @@ def train(
                     trackers_to_train, domain, interpreter=interpreter, **kwargs
                 )
 
-            self.action_fingerprints = rasa.core.training.training.create_action_fingerprints(
+            training = rasa.core.training.training
```

Comment:
I find this a bit confusing. I'd rather do a `from rasa.core.training import training` in this case.